.github/workflows/test_ci.yml

```diff
@@ -0,0 +1,91 @@
+name: Test
+
+on:
+  push:
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        include:
+          - name: ubuntu_16_04-x86_64
+            os: ubuntu-16.04
+            TARGET_CPU: x86-64
+
+          - name: ubuntu_18_04-x86_64
+            os: ubuntu-18.04
+            TARGET_CPU: x86-64
+
+          - name: ubuntu_20_04-x86_64
+            os: ubuntu-20.04
+            TARGET_CPU: x86-64
+
+          - name: osx_11_0-x86_64
+            os: macos-11.0
+            TARGET_CPU: x86-64
+
+          - name: osx_10_11-x86_64
+            os: macos-10.15
+            TARGET_CPU: x86-64
+
+          - name: win-x86_64
+            os: windows-2019
+            TARGET_CPU: x86-64
+
+          - name: ubuntu_16_04-x86_64-haswell
+            os: ubuntu-16.04
+            TARGET_CPU: haswell
+
+          - name: ubuntu_18_04-x86_64-haswell
+            os: ubuntu-18.04
+            TARGET_CPU: haswell
+
+          - name: ubuntu_20_04-x86_64-haswell
```

Comment:
The only haswell targets we want are for 64-bit windows and ubuntu_18_04-x86_64-haswell. We're deprecating them, so we don't want to add more package variants to maintain.